extensions/amp-analytics/0.1/vendors/rudderstack.json

```diff
@@ -0,0 +1,39 @@
+{
+  "transport": {
+    "beacon": false,
+    "xhrpost": false,
+    "image": true
```

Comment:
AMP-analytics will try beacon -> xhrpost -> image requests and handle the fallback. Just to confirm, your servers only accept image requests?

Reply:
Yes, our servers only accept image requests.